### ISSUE TITLE
Avoid errors when upgrading SCA policies

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#3595]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.10.0 | SCA | Manager/Agent | All | All |

The SCA configuration includes the policies to scan by default. That provokes errors when the policies set in the configuration file are missing, and limits us to modify policies between versions without avoiding errors when a policy is deleted or added.

We have to include a solution to make the SCA configuration transparent to the default policies found in the default directory (`ruleset/SCA`).

**Comment by chemamartinez** [comment]
These errors will be shown when upgrading the agent:

```
2019/06/28 07:21:16 sca: WARNING: Policy file not found: '/var/ossec/ruleset/sca/cis_debian_linux_rcl.yml'. Skipping it.
2019/06/28 07:21:16 sca: WARNING: Policy file not found: '/var/ossec/ruleset/sca/system_audit_rcl.yml'. Skipping it.
2019/06/28 07:21:16 sca: WARNING: Policy file not found: '/var/ossec/ruleset/sca/system_audit_ssh.yml'. Skipping it.
2019/06/28 07:21:16 sca: WARNING: Policy file not found: '/var/ossec/ruleset/sca/system_audit_pw.yml'. Skipping it.
```

**Comment by chemamartinez** [comment]
Merged #3607
